bin_tests/src/lib.rs

```diff
@@ -1,8 +1,11 @@
 // Copyright 2023-Present Datadog, Inc. https://www.datadoghq.com/
 // SPDX-License-Identifier: Apache-2.0
 
+mod constants;
 pub mod modes;
 
+pub use constants::*;
```

Comment:
I don't really care either way, but any reason that `modes` is `pub mod`, while `constants` is `pub use *`?